### DISCUSSION TITLE
Question: how do i disable hardware acceleration?

### DISCUSSION THREAD

**Discussion by Blukezz** [dnschneid/crouton#5019] [Q&A]
im quite new to crouton and i was trying to run an game through wine and it was extremely choppy and slow, i found online which the fix is to disable hardware acceleration

**Comment by CroutonIsFun** [comment]
I don't think hardware acceleration is enabled by default. There used to be an attempt to enable it, but this was removed quite awhile back because it caused problems. Sorry you're having problems.
